src/main/java/com/artozersky/HackerNewsAPI/controller/PostController.java

```diff
@@ -46,16 +45,12 @@ public ResponseEntity<PostResponseDTO> createPost(@RequestBody @Validated PostCr
     @Override
     public ResponseEntity<List<PostResponseDTO>> getAllPosts() {
 
-        List<NewsPostModel> posts = postService.getAllPosts();
+        List<PostResponseDTO> posts = postService.getAllPosts();
```

Comment:
Please read about pagination and use it instead of lists in all the API, explain why pagination is good in our case.